### ISSUE TITLE
Mi Smart kettle

### ISSUE THREAD

**Issue by Patrick1610** [aholstenson/miio#226]
Could support for the smart kettle be added? Regards Patrick

**Comment by jghaanstra** [comment]
No. The Mi Smart Kettle is a Bluetooth device while this Homey app only has support for WiFi devices.

**Comment by yongman** [comment]
Does miio can support bluetooth gateway devices integration?
